src/m365/spp/commands/model/model-list.ts

```diff
@@ -0,0 +1,77 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import { odata } from '../../../../utils/odata.js';
+import { spp, SppModel } from '../../../../utils/spp.js';
+import { urlUtil } from '../../../../utils/urlUtil.js';
+import { validation } from '../../../../utils/validation.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  siteUrl: string;
+}
+
+class SppModelListCommand extends SpoCommand {
+  public get name(): string {
+    return commands.MODEL_LIST;
+  }
+
+  public get description(): string {
+    return 'Retrieves the list of unstructured document processing models';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['AIBuilderHybridModelType', 'ContentTypeName', 'LastTrained', 'UniqueId', 'PublicationType'];
```

Comment:
PublicationType is a number for my models. What does this actually represent? Is it worth it to include numbers into the default properties?

Reply:
@milanholemans  Ok, I have checked it. At the beginning, I thought it was information if the model is published or not. However, I created some new models, and all of them have 0. So, I simply removed this property